languages/python/concepts/enums/about.md

```diff
@@ -0,0 +1,111 @@
+In Python, [an enum][enum-docs] is a set of unique names that are bound unique, **constant** values. Enums are defined by inheriting an `Enum` class. Built-in enum types are available in the module `enum` and the class `Enum` can be imported using `from enum import Enum`.
+
+```python
+class Color(Enum):
+    RED = 1
+    GREEN = 2
+```
+
+Note that the values of the enum members can be any data types such as str, tuple, float, etc.
+
+```python
+class Color(Enum):
+    RED = 'red'
+    GREEN = 'green'
+```
+
+Enums can also be created via the following [functional API][enum-functional-api].
+
+```python
+Animal = Enum('Animal', 'ANT BEE CAT DOG')
+list(Animal)
+#=> [<Animal.ANT: 1>, <Animal.BEE: 2>, <Animal.CAT: 3>, <Animal.DOG: 4>]
+
+Animal.ANT.value
+#=> 1
+```
+
+When assigning the same value to two members in an enum, the latter assigned member will be an alias to the formed one. It is not allowed to use the same name for two members of an enum.
+
+```python
+class Color(Enum):
+    RED = 1
+    GREEN = 2
+    ALIAS_OF_RED = 1
+
+Color.ALIAS_OF_RED
+#=> <Color.RED: 1>
+```
+
+Iterating through the members of the enum can be done with:
```

Comment:
```suggestion
Iterating through the members of the enum can be done with the standard `for member in` stynatx:
```

Reply:
That should be “syntax”... I think.

Reply:
WOW.  I don't think "dyslexia" covers that...that looks like I had the keyboard set to a RTL language....or _something_.  Ooof.  Thank you for the catch, @yawpitch!